src/cmd-aws-replicate

```diff
@@ -68,19 +68,31 @@ def run_ore():
     if args.log_level:
         ore_args.extend(['--log-level', args.log_level])
     ore_args.extend(['aws', 'copy-image', '--image', source_image, '--region', source_region])
-    ore_args.extend(region_list)
-    print("+ {}".format(subprocess.list2cmdline(ore_args)))
-    ore_data = json.loads(subprocess.check_output(ore_args))
-    # This matches the Container Linux schema:
-    # https://stable.release.core-os.net/amd64-usr/current/coreos_production_ami_all.json
-    ami_data = [{'name': region,
-                 'hvm': vals['ami'],
-                 'snapshot': vals['snapshot']}
-                for region, vals in ore_data.items()]
-    buildmeta['amis'].extend(ami_data)
+
+    upload_failed_in_region = ""
```

Comment:
If we `continue`, then feels like this should be a list instead, right?

Reply:
Right, pushed a commit with `break` instead